src/System.Text.Json/tests/Serialization/PropertyVisibilityTests.cs

```diff
@@ -216,5 +216,94 @@ public ClassWithIgnoreAttributeProperty()
             [JsonIgnore]
             public string[] MyStringsWithIgnore { get; set; }
         }
+
+        private enum MyEnum
+        {
+            Case1 = 0,
+            Case2 = 1,
+        }
+
+        private struct ClassWithOverride
```

Comment:
nit: This should be StructWithOverride

```suggestion
        private struct StructWithOverride
```

Reply:
Yep. Thanks